katas/metal-slug/jeanpeare/Character.js

```diff
@@ -0,0 +1,61 @@
+export class Character {
+    constructor (CharacterType, Health) {
+        this.CharacterType = CharacterType;
+        this.Health = Health;
+    }
+}
+
+export const Civilian = new Character (
+    'Civilian',
+    1);
+
+export class Player extends Character{
+    constructor(CharacterType, Health, Life){
+        super(CharacterType, Health);
+        this.Life = Life;
+    }
+}
+
+export const Player1 = new Player(
+    'Player',
+    1,
+    3);
+
+export class Enemy extends Character{
+    constructor(CharacterType, Health, Armor, Abilitie){
+        super(CharacterType, Health);
+        this.Armor = Armor;
+        this.Abilitie = Abilitie;
+    }
+}
+
+export const Soldier = new Enemy(
```

Comment:
The creation of `Enemy` instances are correct, but what would be the purpose of creating them here? maybe those should be created in tests instead, and add an expectation of what they are expected to do or have